be/src/storage/txn_manager.cpp

```diff
@@ -49,6 +50,12 @@ TxnManager::TxnManager(int32_t txn_map_shard_size, int32_t txn_shard_size)
     _txn_map_locks = std::unique_ptr<std::shared_mutex[]>(new std::shared_mutex[_txn_map_shard_size]);
     _txn_tablet_maps = std::unique_ptr<txn_tablet_map_t[]>(new txn_tablet_map_t[_txn_map_shard_size]);
     _txn_partition_maps = std::unique_ptr<txn_partition_map_t[]>(new txn_partition_map_t[_txn_map_shard_size]);
+    auto st = ThreadPoolBuilder("meta-flush")
```

Comment:
Use underscore instead of the horizontal line.
```suggestion
    auto st = ThreadPoolBuilder("meta_flush")
```

Reply:
i think use '-' in thread name is more common.